File: home/src/main/java/com/jd/home/viewhodler/ListViewHolder.java
package com.jd.home.viewhodler;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.jd.core.base.adapter.BaseViewHolder;
import com.jd.home.R;

import java.util.Map;

public class ListViewHolder implements BaseViewHolder {
    ImageView imageView;
    TextView textView1;
    TextView textView2;

    @Override
    public int layout_id() {
        return R.layout.home_list_item;
    }

    @Override
    public void onCreateView(View view) {
        imageView = (ImageView) view.findViewById(R.id.image1);
        textView1 = (TextView) view.findViewById(R.id.text1);
        textView2 = (TextView) view.findViewById(R.id.text2);
    }

    @Override
    public void onStart(Object data) {
        Map<String,Object> item = (Map<String,Object>)data;
//        imageView.setImageResource(Datas.get(i).getImageId());
        textView1.setText(item.get("title").toString());
    }
}

File: JDApp-Kotlin/home/src/main/java/com/jd/home/viewhodler/ListViewHolder.kt
package com.jd.home.viewhodler

import android.view.View
import android.widget.ImageView
import android.widget.TextView

import com.jd.core.base.adapter.BaseListViewAdapter
import com.jd.home.R

class ListViewHolder : BaseListViewAdapter.BaseViewHolder {
    lateinit internal var imageView: ImageView
    lateinit internal var textView1: TextView

    override fun layout_id(): Int {
        return R.layout.home_fragment_list_item
    }

    override fun onCreateView(view: View?) {
        imageView = view!!.findViewById<View>(R.id.list_item_image) as ImageView
        textView1 = view.findViewById<View>(R.id.list_item_title) as TextView
    }

    override fun onStart(data: Any?) {
        val item = data as Map<String, Any>?
        //        imageView.setImageResource(Datas.get(i).getImageId());
        textView1.text = item!!["title"]!!.toString()
    }
}
